src/azure-cli/azure/cli/command_modules/acr/_utils.py

```diff
@@ -507,11 +505,14 @@ def create_default_scope_map(cmd,
             raise CLIError('The default scope map was already configured with different repository permissions.' +
                            '\nPlease use "az acr scope-map update -r {} -n {} --add <REPO> --remove <REPO>" to update.'
                            .format(registry_name, scope_map_name))
-    except CloudError:
+    except ResourceNotFoundError:
         pass
     logger.info('Creating a scope map "%s" for provided permissions.', scope_map_name)
-    poller = scope_map_client.create(resource_group_name, registry_name, scope_map_name,
-                                     actions, scope_map_description)
+    scope_map_request = {
+        'actions': actions,
+        'scope_map_description': scope_map_description
+    }
+    poller = scope_map_client.begin_create(resource_group_name, registry_name, scope_map_name, scope_map_request)
```

Comment:
Can you explain why is this change needed? There wasn't a change in the SDK as far as I know.

Reply:
I see this pattern repeat I assume is part of the track2 SDK change. But can you provide some context, please?

Reply:
I am assuming they are using kwargs to make method calls more concise and less prone to error. Positional method calls can be risky if order of calls is incorrect or changed. named arguments / kwargs allows for specifying what value maps to what argument. 
This is just a guess and hopefully @00Kai0 can clarify

Reply:
Refer to this [migrate guide](https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/containerregistry/azure-mgmt-containerregistry/azure/mgmt/containerregistry/v2020_11_01_preview/operations/_scope_maps_operations.py#L179)

Reply:
track 2 uses unflattened parameters instead of flattened parameters (track 1)